src/server/db_slice.cc

```diff
@@ -376,7 +376,7 @@ void DbSlice::AutoUpdater::Run() {
   DCHECK(fields_.action == DestructorAction::kRun);
   CHECK_NE(fields_.db_slice, nullptr);
 
-  fields_.db_slice->PostUpdate(fields_.db_ind, fields_.it, fields_.key, fields_.key_existed);
+  fields_.db_slice->PostUpdate(fields_.db_ind, fields_.it, fields_.key, fields_.orig_heap_size);
```

Comment:
I am not sure that the accounting for memory works well when we use the api of AddNew, if we move the object from another key.
The flows is that we call std::move from source value 
than we call add New which will say that the original heap size is 0 because we dont find the new key
than when we call post update we will add the heap size of the moved object but I belive when we will call del on the souce key we will not remove the heap size we moved the object.
It was not intrudocued in this PR it just came to me now when reviewing this one

Reply:
I just checked (with `RENAME`) and the memory before and after seems to be the same in all of my tests...
Also logically, the `FindMutable()` for source should run `PostUpdate()` with new-size==0 (because it's an empty object), while the `PostUpdate()` for dest should add the real value, so I'm not sure what is the issue?

Reply:
ok I guess there is no issue, I just got confused there